dispatcher/src/main/java/io/zeebe/dispatcher/DispatcherBuilder.java

```diff
@@ -49,8 +49,8 @@ public DispatcherBuilder name(final String name) {
    * The number of bytes the buffer is be able to contain. Represents the size of the data section.
    * Additional space will be allocated for the meta-data sections
    */
-  public DispatcherBuilder bufferSize(final ByteValue byteValue) {
-    bufferSize = (int) byteValue.toBytes();
+  public DispatcherBuilder bufferSize(final long bufferSize) {
+    this.bufferSize = (int) bufferSize;
```

Comment:
Why not have the argument be an int to make this cast explicit to the caller (to avoid any potential confusion)?

Reply:
Done